Update and extend lazer mod articles (#14794)
ppy/osu-wiki: wiki/Gameplay/Game_modifier/Hold_Off/en.md

**Note:** For the full list of all [lazer](/wiki/Client/Release_stream/Lazer) mods, see [Game modifier (lazer)](/wiki/Gameplay/Game_modifier_(lazer))
:::

> The **Hold Off** mod removes all [hold notes](/wiki/Beatmap/Pattern/osu!mania/Hold_note) from a [beatmap](/wiki/Beatmap) and replaces them with normal notes. Specifically, for each hold note, a normal note is placed at its beginning and the hold note is removed entirely.

Reviewer: ```suggestion
The **Hold Off** mod removes all [hold notes](/wiki/Beatmap/Pattern/osu!mania/Hold_note) from a [beatmap](/wiki/Beatmap) and replaces their heads and tails with regular notes.
```
less repetition

Author: Sounds better, but is not correct. Only the heads are replaced.

```suggestion
The **Hold Off** mod removes all [hold notes](/wiki/Beatmap/Pattern/osu!mania/Hold_note) from a [beatmap](/wiki/Beatmap) and replaces their heads with regular notes.
```

Reviewer: yeah misread this one
